app/src/main/java/com/main/mealplanner/dto/RecipeList.kt

```diff
@@ -2,6 +2,13 @@ package com.main.mealplanner.dto
 
 import com.google.gson.annotations.SerializedName
 
-data class RecipeList(@SerializedName("meals") val meals: ArrayList<RecipeHeader>){
+data class RecipeList(@SerializedName("meals") val meals: ArrayList<RecipeHeader>) {
+
+    /***
+     * overrides default toString method
+     */
+    override fun toString(): String {
```

Comment:
SOLID standard to always override the toString method in case it is needed later